modules/web/js/ballerina/model/tree-util.js

```diff
@@ -408,6 +408,10 @@ class TreeUtil extends AbstractTreeUtil {
             // Get the parent node.
             const expressionParentNode = node.parent;
 
+            source = source.substr(source.length - 1) === ';'
```

Comment:
We can use endsWith function here.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith